src/librustc_typeck/diagnostics.rs

```diff
@@ -380,6 +380,21 @@ fn main() {
 ```
 "##,
 
+E0044: r##"
+You can't use type parameters on foreign items. Example of erroneous code:
+
+```
+extern { fn some_func<T>(x: T); }
+```
+
+To fix this, replace the type parameter with the specializations that you
+need:
+
+```
+extern { fn some_func_i32(x: i32); }
+extern { fn some_func_i64(x: i64); }
+```
+
```

Comment:
You're missing an end of string here, currently E0045 and its message are included in E0044's message!